src/python_testing/TC_CADMIN_1_22.py

```diff
@@ -115,48 +122,17 @@ async def test_TC_CADMIN_1_22(self):
         asserts.assert_equal(window_status2, Clusters.AdministratorCommissioning.Enums.CommissioningWindowStatusEnum.kWindowNotOpen,
                              "Commissioning window is expected to be closed, but was found to be open")
 
-    def pics_TC_CADMIN_1_24(self) -> list[str]:
-        return ["CADMIN.S"]
-
-    def steps_TC_CADMIN_1_24(self) -> list[TestStep]:
-        return [
-            TestStep(1, "Commissioning, already done", is_commissioning=True),
-            TestStep(2, "TH_CR1 opens a commissioning window on DUT_CE using ECM with a value of 180 seconds",
-                     "DUT_CE opens its Commissioning window to allow a second commissioning"),
-            TestStep(3, "Wait for the commissioning window in step 2 to timeout"),
-            TestStep(4, "TH_CR1 reads the window status to verify the DUT_CE window is closed",
-                     "DUT_CE windows status shows the window is closed"),
-            TestStep(5, "TH_CR1 opens a commissioning window on DUT_CE using ECM with a value of 179 seconds",
-                     "DUT_CE does not open its Commissioning window to allow a second commissioning. DUT_CE shows 'Failed to open commissioning window. Global status 0x85'"),
-            TestStep(6, "TH_CR1 reads the window status to verify the DUT_CE window is closed",
-                     "DUT_CE windows status shows the window is closed"),
-        ]
-
-    @async_test_body
-    async def test_TC_CADMIN_1_24(self):
-        self.step(1)
-        self.th1 = self.default_controller
-        self.discriminator = random.randint(0, 4095)
-
-        self.step(2)
+        self.step(7)
         await self.th1.OpenCommissioningWindow(
             nodeid=self.dut_node_id, timeout=180, iteration=10000, discriminator=self.discriminator, option=1)
 
-        self.step(3)
-        sleep(180)
-
-        self.step(4)
-        # TODO: Issue noticed when initially attempting to check window status after waiting for timeout to occur, issue is detailed here: https://github.com/project-chip/connectedhomeip/issues/35983
-        # Workaround in place below until above issue resolved
-        try:
-            window_status = await self.support.get_window_status(th=self.th1)
-        except asyncio.CancelledError:
-            window_status = await self.support.get_window_status(th=self.th1)
-
-        asserts.assert_equal(window_status, Clusters.AdministratorCommissioning.Enums.CommissioningWindowStatusEnum.kWindowNotOpen,
-                             "Commissioning window is expected to be closed, but was found to be open")
+        self.step(8)
+        revokeCmd = Clusters.AdministratorCommissioning.Commands.RevokeCommissioning()
+        await self.th1.SendCommand(nodeid=self.dut_node_id, endpoint=0, payload=revokeCmd, timedRequestTimeoutMs=6000)
+        # The failsafe cleanup is scheduled after the command completes, so give it a bit of time to do that
+        sleep(1)
```

Comment:
Does it work if you use the get_window_status function here rather than a hard-coded sleep?